torch/fx/experimental/proxy_tensor.py

```diff
@@ -240,7 +240,7 @@ def inner(e):
 def fetch_tensor_proxy(tracer):
     return lambda t: get_proxy_slot(t, tracer, t)
 
-HANDLED_TYPES = (torch.Tensor, torch.nn.Parameter)
+HANDLED_TYPES = (torch.Tensor, torch.nn.Parameter, torch._subclasses.fake_tensor.FakeTensor)
```

Comment:
@jansel @ezyang This line causes an interesting regression. It adds a bunch of unsued `detached` tensors that are not used

As an example, `test_simple`

```python
    def test_simple(self):
        def f(x):
            # simple test: 1 view op, 1 inplace op
            tmp = torch.ones(4, 2)
            y = x.view(4, 2)
            y.add_(tmp)
            z = x * x
            return y
        self.assert_functionalization(f, torch.ones(4, 2))
        logs = self.get_logs(f, torch.ones(4, 2))
        self.assertExpectedInline(logs, """\



def forward(self, arg0_1):
    ones = torch.ops.aten.ones.default([4, 2], device = device(type='cpu'), pin_memory = False)
    view_copy = torch.ops.aten.view_copy.default(arg0_1, [4, 2])
    add = torch.ops.aten.add.Tensor(view_copy, ones);  view_copy = ones = None
    view_copy_1 = torch.ops.aten.view_copy.default(add, [4, 2]);  add = None
    view_copy_2 = torch.ops.aten.view_copy.default(view_copy_1, [4, 2])
    mul = torch.ops.aten.mul.Tensor(view_copy_1, view_copy_1)
    copy_ = torch.ops.aten.copy_.default(arg0_1, view_copy_1);  arg0_1 = view_copy_1 = None
    return view_copy_2
    """)

changed to 

```python
def forward(self, arg0_1):
    ones = torch.ops.aten.ones.default([4, 2], device = device(type='cpu'), pin_memory = False)
    _param_constant0 = self._param_constant0
    detach = torch.ops.aten.detach.default(_param_constant0);  _param_constant0 = None
    view_copy = torch.ops.aten.view_copy.default(arg0_1, [4, 2])
    _param_constant1 = self._param_constant1
    detach_1 = torch.ops.aten.detach.default(_param_constant1);  _param_constant1 = None
    _param_constant2 = self._param_constant2
    detach_2 = torch.ops.aten.detach.default(_param_constant2);  _param_constant2 = None
    add = torch.ops.aten.add.Tensor(view_copy, ones);  view_copy = ones = None
    view_copy_1 = torch.ops.aten.view_copy.default(add, [4, 2]);  add = None
    view_copy_2 = torch.ops.aten.view_copy.default(view_copy_1, [4, 2])
    _param_constant3 = self._param_constant3
    detach_3 = torch.ops.aten.detach.default(_param_constant3);  _param_constant3 = None
    _param_constant4 = self._param_constant4
    detach_4 = torch.ops.aten.detach.default(_param_constant4);  _param_constant4 = None
    mul = torch.ops.aten.mul.Tensor(view_copy_1, view_copy_1)
    copy_ = torch.ops.aten.copy_.default(arg0_1, view_copy_1);  arg0_1 = view_copy_1 = None
    return view_copy_2
    """)
```

As can be seen, a bunch of 

```python
    _param_constantXXX = self._param_constantXXX
    detach = torch.ops.aten.detach.default(_param_constantXXX);  _param_constantXXX = None
```

show up. Any idea how to get around this?